beastling/beastxml.py

```diff
@@ -104,11 +104,14 @@ def add_beastling_comment(self):
         BEASTling version used to create the file, the time and date of
         generation and the original configuration file text.
         """
+        from clldutils.inifile import INI
```

Comment:
I think I copied this in here to lookup some attribute of `INI` from pycharm :) and then forgot to take it out again ...